<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сапёр</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="topBar">
      <item>
       <widget class="QComboBox" name="levelComboBox">
        <item>
         <property name="text">
          <string>Лёгкий</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Средний</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Высокий</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string>Время: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newGameButton">
        <property name="text">
         <string>Новая игра</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveGameButton">
        <property name="text">
         <string>Сохранить игру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadGameButton">
        <property name="text">
         <string>Загрузить игру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showRatingsButton">
        <property name="text">
         <string>Показать рейтинги</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="gameBoard">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
